### PR TITLE
Add Danish localization

### PR DESCRIPTION
**Pull request by heidipuk**
Add Danish localisation to the Vapor website

### CODE REVIEW

File: src/i18n/dk.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        slogan: 'Fremtidens webudvikling'
+    },
+    callToAction: {
+        start: 'Kom igang',
+        join: 'Join Chat'
+    },
+    nav: {
+        code: 'Kode',
+        cloud: 'Cloud',
+        docs: 'Dokumenter'
```
Comment (joscdk) [inline src/i18n/dk.js]: I would maybe call this `Dokumentation`

File: src/i18n/dk.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    bio: {
+        first: 'Ikke-blokerende, event-dreven arkitektur bygget oven på Apple\'s <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> leverer Vapor <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">høj ydeevne</a>.',
+        second: 'Skrevet i <a target="_blank" href="https://swift.org">Swift</a>, det kraftfulde programmeringssprog, der samtidig er nemt at lære.',
+        third: 'Ekspressivt, <a target="_blank" href="https://developer.apple.com/videos/play/wwdc2015/408/">protokol-orienteret design,</a> med fokus på typesikkerhed og vedligeholdelse.'
+    },
+    contribute: {
+        title: 'Bidrag!',
+        options: {
+            code: {
+                description: 'Indsend forslag til nye funktioner, reporter fejl, og merge kode på Github!'
```
Comment (joscdk) [inline src/i18n/dk.js]: Maybe this should be something like this: `Deltag i udviklingen, foreslå nye features eller reporter fejl på Github!`

**Review by joscdk — CHANGES_REQUESTED** [review]
@heidipuk thank you for this.. I have suggested a couple of alternatives

**Review by joscdk — APPROVED** [review]

**Comment by tanner0101** [comment]
Thank @heidipuk for submitting this! And thanks @joscdk for the third party review. Merging :)

**Comment by penny-coin** [comment]
Hey @heidipuk, you just merged a pull request, have a coin! 

You now have 1 coins.

**Pull request merged**